### DISCUSSION TITLE
Utilization - Prefixes

### DISCUSSION THREAD

**Discussion by milalima** [netbox-community/netbox#6216] [Q&A]
Hi guys, I am newly working wiht Netbox development and I have a doubt. 

My installation is not showing the utilization:

<img width="556" alt="image" src="https://user-images.githubusercontent.com/48295637/115313657-878f2080-a128-11eb-9274-058c40b7c011.png">

What should I do in this case?

My version is 2.8.7

**Comment by candlerb** [comment]
Do you have child prefixes underneath those prefixes? Are they in the global VRF?

Can you try changing the status to "Container"?  IIRC, a prefix which contains other prefixes should be "Container" (and the utilization is calculated as the % of space allocated to child prefixes); a prefix which contains direct IP address children is "Active"

However, I don't have an old version of Netbox lying around to test this with.  Here's how 2.11.0 looks at `/ipam/prefixes/`:

![image](https://user-images.githubusercontent.com/44789/115349476-cf0ac080-a1ab-11eb-84c4-8df58410685d.png)

**Reply by milalima** [reply]
Hi @candlerb you were right, the issue here was the status wasn't as container. As soon I changed, the Utilization starts to appear. Thanks for the tip.
